handbook/product/index.md

```diff
@@ -20,6 +20,56 @@ See job descriptions and responsibilities of roles on the Product team:
 
 See "[Personas](personas.md)".
 
+## Delivery plans
+
+Every customer-facing piece of work at Sourcegraph has a **delivery plan**, which describes how we deliver the work to customers. Delivery plans have 3 purposes:
+
+- They let us ship things to customers more quickly without blocking on any other team member or to relay context. Usually, this means developers notify customers directly and can get immediate feedback on their work.
+- They are an easily verifiable check during planning that all work ties directly to a customer need that is understood by the developers working on the feature.
+- They give developers more direct contact with customers (which is motivating and educational).
+
+Delivery plans live in the [sourcegraph/customer issue tracker](https://github.com/sourcegraph/customer/issues), which is visible only to Sourcegraph team members.
+
+### Format
+
+There is no strict format for delivery plans. (This may change as we use them more.)
+
+We anticipate most delivery plans will:
+
+- Have a series of steps that the developer can follow on their own without blocking on any other team member (e.g., sales or distribution). Usually this will be an email or Slack message to send to 1 or 2 specific customers with docs links, screenshots, and/or example links.
+- Have hypotheses to help us learn more quickly. There are 2 kinds of hypotheses. These help us learn quickly if we've misjudged the importance or ease of a feature.
+  - Example hypothesis 1: Testing whether a feature is actually important to a customer. The hypothesis is of the form `Within X days, Alice ____` (e.g., "Within 3 days, Alice replies to the email and uses the feature").
+  - Example hypothesis 2: Testing whether a feature works as expected. The hypothesis is of the form `Alice is able to ____` (e.g., "Alice is able to perform an interactive-mode search").
+- Be written together by team members on Product/Sales/Engineering/Marketing (depending on the feature).
+  - It is not the developer's responsibility in general to know what the delivery plan should be or to write it themselves. That information is crucial context that usually comes from Product or Sales.
+
```

Comment:
It would be great to also track success/failure and look back on how well the plan went so we can continue to get better.

```suggestion
- Have a conclusions and results of hypotheses and plans
   - It is important that we learn from the successes and failures of these plans

```

Reply:
Good point. Added to the process section instead.